src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java

```diff
@@ -60,6 +60,7 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(POST, "/subscriptions/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(GET, "/subscriptions/**").access(hasScope(eventStreamReadScope))
```

Comment:
`/subscriptions/* instead of /subscriptions/**`
I think there was a problem with that
1) if you make it '/subscriptions/*' - it will match only URLs with exactly one path segment after /subscriptions/
2) If you make it '/subscriptions' - it will mot match the URL ending with slash
3) If you make it '/subscriptions/' - it will not allow URL without slash at the end
:(
But I understand what you don't like here. But let's separate this discussion/refactoring as a separate task.